internal/util.ml

```diff
@@ -3602,3 +3602,7 @@ let rec filter_map fn = function
 let rec rev_iter fn = function
   | [] -> ()
   | hd :: tl -> let () = rev_iter fn tl in fn hd
+
+let print_version_commit =
```

Comment:
You miss a `()` parameter. This version will print and exit as soon as you launch any command.